src/components/other/other.components.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Header from "./header.other";
+
+function Other() {
+  return (
+    <>
+      <Header />
+    </>
```

Comment:
```suggestion
      <Header />
```

굳이 `fragment`를 넣을 필요는 없을 것 같아용